contrib/python/create_slab2_database.ipynb

```diff
@@ -0,0 +1,267 @@
+{
```

Comment:
Can you set this up so that the script automatically places the output data file in `data/initial_composition/slab`? You could also move the script over there.